### DISCUSSION TITLE
API RPC issues

### DISCUSSION THREAD

**Discussion by cryptocrier** [pwillworth/dfkreport#26] [Q&A] [answered]
I am trying to collect all my info across all the chains for DFK but i keep getting the same rpc issue. I have had other people try it in their browsers as well and it is saying the same thing. stuck at 5% and won't move past that. any ideas?
![image](https://user-images.githubusercontent.com/97005778/210003682-dc8118bf-1d82-4b0f-94e2-553f1f791278.png)

**Comment by cryptocrier** [comment]
0x9b1feEA748Be8d0d787b73A23ae7D432DA8FDD2E

**Comment by pwillworth** [comment, marked as answer]
Ok, we were having some more timeout issues with one of the providers I use.  I added some more handling to that and tested a report for your report and it completed.  If you want to try it again, you should be good.
